src/tiled/mapdocument.h

```diff
@@ -319,35 +315,22 @@ public slots:
     void deselectObjects(const QList<MapObject*> &objects);
     void moveObjectIndex(const MapObject *object, int count);
 
-    QString mLastExportFileName;
-
     /*
      * QPointer is used since the formats referenced here may be dynamically
      * added by a plugin, and can also be removed again.
      */
     QPointer<MapFormat> mReaderFormat;
     QPointer<MapFormat> mWriterFormat;
-    QPointer<MapFormat> mExportFormat;
     Map *mMap;
     LayerModel *mLayerModel;
     QRegion mSelectedArea;
     QList<MapObject*> mSelectedObjects;
     MapRenderer *mRenderer;
     Layer* mCurrentLayer;
     MapObjectModel *mMapObjectModel;
+    QPointer<MapFormat> mExportFormat;
```

Comment:
Why move this pointer to the end?

Reply:
Fixed

Not sure how I moved it down there... I suspect I must have removed it and then added it as a new variable as I always add new variables at the end.